Add readme file for promotional material

jancborchardt opened the pull request:
Just a bunch of info for the logo, sticker and presentation. :)

Please review @camilasan @Incabell @sleepypioneer @evalica @jonatoni

evalica commented on promo/README.md:
```diff
@@ -0,0 +1,62 @@
+
+## 🌼 Logo
+
+![](sticker-45mm.jpg)
```
Maybe we should add some width / height to not look this big?

evalica commented on promo/README.md:
```diff
@@ -0,0 +1,62 @@
+
+![](sticker-45mm.jpg)
+
+The logo is white on `#45b086` green/turquoise. If you need the logo on white, use the inverse. There are more logo versions in the `img` folder.
```
You could have put `#fff` (white) :)

evalica replied:
`img` could use a link?

evalica commented on promo/README.md:
```diff
@@ -0,0 +1,62 @@
+
+The logo is white on `#45b086` green/turquoise. If you need the logo on white, use the inverse. There are more logo versions in the `img` folder.
+
+As typeface we use [Nunito](https://github.com/opensourcediversity/opensourcediversity.org/tree/master/css) in Regular and Bold. (No other weights and no italic styles.)
```
The punctuation here is bad :) but minor

evalica commented on promo/README.md:
```diff
@@ -0,0 +1,62 @@
+
+![](sticker-mockup.png)
+
+We usually order [Flyeralarm outdoor stickers](https://www.flyeralarm.com/de/shop/configurator/index/quantity/7983970#159=601&160=602&161=615&162=585), 4.5 cm round. If you want to order some, use the file `sticker-45mm.jpg` for printing.
```
Link on  `sticker-45mm.jpg` ?

evalica commented on promo/README.md:
```diff
@@ -0,0 +1,62 @@
+
+![](banner-presentation.png)
+
+We have the `Open Source Diversity presentation.pdf` which is roughly 20 minutes long, and `Open Source Diversity presentation lightning.pdf` for 5-minute lightning talks.
```
Link on `Open Source Diversity presentation.pdf` and `Open Source Diversity presentation lightning.pdf`?

evalica commented on promo/README.md:
```diff
@@ -0,0 +1,62 @@
+We have the `Open Source Diversity presentation.pdf` which is roughly 20 minutes long, and `Open Source Diversity presentation lightning.pdf` for 5-minute lightning talks.
+
+Anyone is encouraged to give the presentation at a conference! 😊 If you do so, feel free to use these texts for submitting the talk:
+
```
Maybe putting the whole part down with the text for the talk in a quote or code container? otherwise is hard to know where it ends.

jancborchardt commented:
Wow, thorough review @evalica! :) Thanks a lot – I fixed all the points you brought up.

Also I added the whole promo readme as part into the main README file as otherwise there will just be confusion as to where what is. :)
